classic/original_autogpt/autogpt/app/twzrd_agent_intel.py

```
@@ -0,0 +1,155 @@
+    risk_flags: list
+    preflight_passed: bool
+    authorized: bool
+
+
+async def _call_mcp_tool(tool_name: str, wallet: str) -> str:
+    """Call a TWZRD Agent Intel MCP tool and return the raw text response."""
+    try:
+        from mcp import ClientSession
+        from mcp.client.streamable_http import streamablehttp_client
+    except ImportError:
+        raise ImportError("mcp package not installed. Run: pip install mcp")
```

Ruff (0.15.15)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)